Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ReflectedPropertyEditor.cpp

```diff
@@ -1356,8 +1384,14 @@ namespace AzToolsFramework
 
         // get the property editor
         auto rowWidget = m_widgets.find(it->second);
-        if (rowWidget != m_widgets.end())
+        auto rowWidgetGroup = m_specialGroupWidgets.find(it->second);
+        if (rowWidget != m_widgets.end() || rowWidgetGroup != m_specialGroupWidgets.end())
```

Comment:
suggestion: A guide comment here might be good to help the reader understand why this is needed

Reply:
Thanks, added it!